demo/build.js

```diff
@@ -0,0 +1,52 @@
+import fs from 'fs';
```

Comment:
This is an old-school build file....
Where's the Grunt/Gulp/Webpack 😄

Reply:
Beautiful isn't it. Crazy to actually be able to see exactly how your build works :grin:

Reply:
I mean... I don't know how the innards of esbuild work...